src/components/core/themeButton/index.tsx

```diff
@@ -28,20 +28,21 @@ export default function ThemeButton({ iconSize }: { iconSize: number }) {
     <>
       {theme === "dark" ? (
         <button
-          className="px-2 py-2 md:flex fixed md:static block md:top-0 md:m-0 top-1 m-2 rounded-full hover:bg-primary-light hover:scale-105 bg-primary text-white"
+          className="px-2 py-2 md:flex fixed md:static block md:top-0 md:m-0 top-1 m-2 rounded-full hover:bg-primary-light hover:scale-105 border-2 border-white md:bg-transparent md:border-2 md:border-primary text-white"
```

Comment:
roll back to original one, as this component might hamper for different screen sizes. You can please remove the fixed property